utilities/configure_scripts/configure_dealii.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+## ---------------------------------------------------------------------
+##
+## PartExa - A Particle Library for the Exa-Scale
+##
+## Copyright (C) 2022 by the PartExa authors
+##
+## This program is free software: you can redistribute it and/or modify
+## it under the terms of the GNU General Public License as published by
+## the Free Software Foundation, either version 3 of the License, or
+## (at your option) any later version.
+##
+## This program is distributed in the hope that it will be useful,
+## but WITHOUT ANY WARRANTY; without even the implied warranty of
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+## GNU General Public License for more details.
+##
+## You should have received a copy of the GNU General Public License
+## along with this program. If not, see <https://www.gnu.org/licenses/>.
+##
+## ---------------------------------------------------------------------
+
+# You may want to deactivate this line
+rm -rf CMakeFiles/ CMakeCache.txt
+
+# It is recommended to copy this file to your deal.II build directory via
+# ```bash
+# cd <dealii_build>
+# cp <partexa_source>/utilities/configure_scripts/configure_partexa.sh .
+# ```
+
+# Having copied the file, you need to adjust the paths specified below
+P4EST=/path/to/p4est
+DEAL=/path/to/dealii
```

Comment:
I think we should move the comment and also the definition of the two paths to the Sphinx documentation here https://partexa.github.io/PartExa-Sphinx/installation/dependencies.html and rewrite the Sphinx documentation to something like:
```bash
mkdir <dealii_build> <dealii_install>
cd <dealii_build>
cp <partexa_source>/utilities/configure_scripts/configure_dealii.sh .
export P4EST=<p4est_install>
export DEAL=<dealii_source>
./configure_dealii.sh
make -j <num_procs> install
```
In this way, the user does not need to open the configure script itself to read the comments.

What I just also noticed, is that the cmake flags given in this script do not set `DCMAKE_INSTALL_PREFIX=<dealii_install>`. I did this so far, I think because it came from the "official" dealii documentation/guides. Would it be okay for you if we keep this flag?

Reply:
Thinking about it again I think we even don't need to copy the script, but just execute it in the build directory.

Reply:
I believe one needs to copy the script, so that the user can make adjustments. The number of parameters will increase, e.g. activation of tests might/should be optional, there will probably be other third party libraries that are optional etc. Also, I want to set the paths only once, and then this script should run for all times on a given system.

In essence, I believe one needs to touch the script. Without a copy, I have the concern that the git workflow becomes impractical when pulling/rebasing.

Put differently, my motivation is to have one single short command in the command line.

Reply:
Ah okay, now I got your point. Let's go with the configure scripts and the user-defined paths! I just had a call with @ppraegla and will shortly post a new comment below with our thoughts.

Reply:
I typically don't use dealii_install